### PR TITLE
Voyages api

### PR DESCRIPTION
**Pull request by ghost**
Authorization is changed!

### CODE REVIEW

**Review by zkwsk — APPROVED** [review]
Looks good! Great work 👯

File: src/server/api/controllers/voyages.controller.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+// to get all the voyages
+const getVoyages = () => {
+  return knex("voyages").select("*");
+};
+
+//to create a voyage
+const createVoyage = async ({ body }) => {
+  try {
+    return knex("voyages").insert({
+      vessel_id: body.vessel_id,
```
Comment (bytewiz) [inline src/server/api/controllers/voyages.controller.js:15]: You could destructure it like, but just an option 🙌 
```
const { vessel_id, ... } = body;
...
return knex("voyages").insert({
  vessel_id,
   ...
```

**Review by bytewiz — APPROVED** [review]
Looks good, nice job! 👍

File: src/server/api/controllers/voyages.controller.js (hunk 1 above)
Comment (ghost) [reply src/server/api/controllers/voyages.controller.js:15]: I will try to keep it in my mind for next time :)

**Pull request merged**